includes/free/class-login.php

```diff
@@ -1091,4 +1079,64 @@ public function default_wp_login_override() {
             exit;
         }
     }
+
+    /**
+     * Send activated email for user
+     *
+     * @param $user
+     */
+    private function send_activated_email( $user ) {
+        $blogname   = wp_specialchars_decode( get_option( 'blogname' ), ENT_QUOTES );
+        $user_email = $user->user_email;
+        $to         = $user_email;
+        $login_url  = wp_login_url();
+
+        if ( wpuf()->is_pro() && wpuf_pro_is_module_active( 'email-templates/email-templates.php' ) && wpuf_get_option( 'account_activated_user_email_notification', 'wpuf_mails', 'on' ) ) {
+            $subject = wpuf_get_option( 'account_activated_user_email_subject', 'wpuf_mails' );
+            $message = wpuf_get_option( 'account_activated_user_email_body', 'wpuf_mails' );
+            $message = self::prepare_mail_body( $message, $user, $blogname );
+        } else {
+            /* translators: %s: blogname */
+            $subject = sprintf( __( '[%s] Account has been activated', 'wp-user-frontend' ), $blogname );
+            /* translators: %s: username */
+            $message  = sprintf( __( 'Hi %s,', 'wp-user-frontend' ), $user->user_login ) . "\r\n\r\n";
+            $message .= __( 'Congrats! Your account has been activated. To login visit the following url:', 'wp-user-frontend' ) . "\r\n\r\n";
+            $message .= $login_url . "\r\n\r\n";
+            $message .= __( 'Thanks', 'wp-user-frontend' );
+        }
+
+        $message = get_formatted_mail_body( $message, $subject );
+
+        $headers = array( 'Content-Type: text/html; charset=UTF-8' );
+
+        $subject  = apply_filters( 'wpuf_mail_after_confirmation_subject', $subject );
+        $message  = apply_filters( 'wpuf_mail_after_confirmation_body', $message );
+
+        wp_mail( $to, $subject, $message, $headers );
+    }
+
+    /**
+     * Preapring mail body for activated email
+     *
+     * @param $message
+     * @param $user
+     * @param $blogname
+     *
+     * @return string|string[]
+     */
+    private static function prepare_mail_body( $message, $user, $blogname ) {
```

Comment:
Do we really need to use a static function here? Unless explicitly necessary, don't make it a static function.